### ISSUE TITLE
Script fails to run

### ISSUE THREAD

**Issue by yellow-starburst** [seajaysec/cypheroth#20]
The following is the Error that I get after running cypheroth:
Address should be of the form: [scheme://][username:password@][host][:port]
Do you know why I get this error @cnotin ?

**Comment by cnotin** [comment]
It looks like it is a Cypher-shell error. Show us your command-line

**Comment by yellow-starburst** [comment]
./cypheroth -u neo4j -p BloodHound -d lab.domain.local

Additionally I am using the newest neo4j and cypher shell version 4.13

**Comment by cnotin** [comment]
Can you try running this command those two commands to compare?
```bash
cypher-shell -u neo4j -p BloodHound -a 127.0.0.1:7687 --format plain "MATCH (n) RETURN CASE WHEN count(n) > 0 THEN 'Connected' ELSE 'Not Connected' END"
cypher-shell -u neo4j -p BloodHound -a neo4j://127.0.0.1:7687 --format plain "MATCH (n) RETURN CASE WHEN count(n) > 0 THEN 'Connected' ELSE 'Not Connected' END"
```

**Comment by yellow-starburst** [comment]
The 2nd command worked.
cypher-shell -u neo4j -p BloodHound -a neo4j://127.0.0.1:7687 --format plain "MATCH (n) RETURN CASE WHEN count(n) > 0 THEN 'Connected' ELSE 'Not Connected' END"

**Comment by yellow-starburst** [comment]
Fix - must specify "-a neo4j://127.0.0.1:7687" in the cyphershell query

**Comment by cnotin** [comment]
Thanks for confirming the issue. It should be fixed with commit above, please try again if you can!
